templates/adding_a_new_model/modeling_xxx.py

```diff
@@ -420,7 +420,7 @@ def forward(
         )
 
         sequence_output = outputs[0]
-        prediction_scores = self.cls(sequence_output)
+        prediction_scores = self.lm_head(sequence_output)
```

Comment:
In XxxForMaskedLM, `self.cls()` does not exist. I think the goal was to call `self.lm_head()` instead. When I created modeling_squeezebert.py from modeling_xxx.py, I inherited this code, and the nonexistance of `self.cls()` is a problem that I found when testing the code.

Reply:
You're correct! Thanks for fixing this!